src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractLanguageService`2.IVsLanguageTextOps.cs

```diff
@@ -26,12 +26,14 @@ internal abstract partial class AbstractLanguageService<TPackage, TLanguageServi
     {
         public int Format(IVsTextLayer textLayer, TextSpan[] selections)
         {
-            var waitIndicator = this.Package.ComponentModel.GetService<IWaitIndicator>();
             var result = VSConstants.S_OK;
-            waitIndicator.Wait(
+            var uiThreadOperationExecutor = this.Package.ComponentModel.GetService<IUIThreadOperationExecutor>();
+            uiThreadOperationExecutor.Execute(
                 "Intellisense",
-                allowCancel: true,
-                action: c => result = FormatWorker(textLayer, selections, c.CancellationToken));
+                defaultDescription: "",
+                allowCancellation: false,
```

Comment:
you changed allowCancel: true to false :(

Reply:
Fixed, good catch!